themes/custom/droopler_theme/scss/components/_d_p_side_embed.scss

```diff
@@ -177,6 +177,12 @@ $d-p-side-embed-desktop-height: auto !default; // You can set fixed height of em
           padding-right: ($grid-gutter-width / 2);
         }
       }
+      &-full {
+        .d-p-side-embed-content-fields {
+          padding-top: 30px;
+          padding-bottom: 30px;
```

Comment:
guys - czy to nie powinno byc w dodane jako zmienna gdzies tam na gorze ?

Reply:
Masz rację, to zrobie tu hotfixa :)